.cargo/common.dic

```diff
@@ -0,0 +1,125 @@
+3
+-
+-
+(
+)
+*
+/
+@dev
+^
++
+<
+=
+>
```

Comment:
Why are these included in the dictionary?

Reply:
Otherwise, we can't use formulas in the comments=)